examples/06_assets/main.rs

```diff
@@ -117,6 +94,10 @@ impl State for Example {
 }
 
 fn main() {
+    let assets_path = format!("{}/examples/06_assets/resources/assets",
+                   env!("CARGO_MANIFEST_DIR"));
+    set_var("AMETHYST_ASSET_DIRS", assets_path);
```

Comment:
I think this should be done by the user, also the "static" set env isn't that great. But I feel like this behaviour should be kept somehow optionally.